.gitignore

```diff
@@ -3,3 +3,4 @@ coverage/
 .vscode
 .env
 dist/bundle.js
+.DS_Store
```

Comment:
This should be *.DS_Store so that we can ignore this file no matter what directory it's in.